src/crypto/crypto_util.h

```diff
@@ -772,7 +772,7 @@ class ArrayBufferOrViewContents {
   T buf = 0;
   size_t offset_ = 0;
   size_t length_ = 0;
-  std::shared_ptr<v8::BackingStore> store_;
+  void* data_ = 0;
```

Comment:
Nit: May I suggest using `nullptr` here?

```suggestion
  void* data_ =  nullptr;
```